src/app/public/modules/description-list/description-list.component.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  AfterContentInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ContentChildren,
+  ElementRef,
+  HostListener,
+  Input,
+  OnDestroy,
+  QueryList,
+  ViewChild
+} from '@angular/core';
+
+import {
+  takeUntil
+} from 'rxjs/operators';
+
+import {
+  Subject
+} from 'rxjs';
+
+import {
+  SkyDescriptionListMode
+} from './types/description-list-mode';
+
+import {
+  SkyDescriptionListOrientation
+} from './types/description-list-orientation';
+
+import {
+  SkyDescriptionListAdapterService
+} from './description-list-adapter-service';
+
+import {
+  SkyDescriptionListContentComponent
+} from './description-list-content.component';
+
+import {
+  SkyDescriptionListService
+} from './description-list.service';
+
+/**
+ * Creates a description list to display term-description pairs.
+ */
+@Component({
+  selector: 'sky-description-list',
+  templateUrl: './description-list.component.html',
+  styleUrls: ['./description-list.component.scss'],
+  providers: [SkyDescriptionListService],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyDescriptionListComponent implements AfterContentInit, OnDestroy {
+
+/**
+ * Specifies a default value to display when no value is provided
+ * for a term-description pair.
+ * @default 'None found'
+ */
+  @Input()
+  public set defaultValue(value: string) {
```

Comment:
Should we rename this property to `defaultDescription`? Before, the `Value` part matched the way we referred to pairs in the definition list as label-value pairs. But now, we are referring to the pairs as term-description pairs to match the terminology the HTML components use. From a functional standpoint, this probably doesn't matter. But from a consistency standpoint, it might. What do you think?

Reply:
Good point. I'll make the change!